### PR TITLE
use use statement instead of namespace

### PR DESCRIPTION
**Pull request by OskarStark**

### CODE REVIEW

File: controller/service.rst [hunk 1]
```diff
@@ -63,9 +63,8 @@ a service like: ``App\Controller\HelloController::index``:
     .. code-block:: php
 
         // config/routes.php
-        namespace Symfony\Component\Routing\Loader\Configurator;
-
         use App\Controller\HelloController;
+        use Symfony\Component\Routing\Loader\Configurator\RoutingConfigurator;
 
         return function (RoutingConfigurator $routes) {
             $routes->add('hello', '/hello')
```
Comment (wouterj) [inline controller/service.rst]: There is a typo one line below this as well: `['HelloController::class`, maybe we can update it directly?

Comment (OskarStark) [reply controller/service.rst]: You mean:
```diff
- ->controller(['HelloController::class, 'index'])
+ ->controller([HelloController::class, 'index'])
```

??

Comment (OskarStark) [reply controller/service.rst]: And I am wondering in the Invokable Controller section which comes next, we are talking about:

`# app/config/routing.yml`, `<!-- app/config/routing.xml -->` and
`// app/config/routing.php`using the `old $collection`

🤔 But this is `4.2` branch

Comment (OskarStark) [reply controller/service.rst]: Typo fixed

**Review by javiereguiluz — APPROVED** [review]

**Review by wouterj — APPROVED** [review]
Looks fine to me, the other one migh be a bit confusing

**Pull request merged**